Source/DataModels/Scene/Scene.cpp

```diff
@@ -605,6 +605,24 @@ GameObject* Scene::SearchGameObjectByID(UID gameObjectID) const
 	return nullptr;
 }
 
+std::vector<GameObject*> Scene::SearchGameObjectByTag(const std::string& gameObjectTag) const
+{
+	std::vector<GameObject*> tagGameObjects;
+	for (int i = 0; GameObject* gameObject : sceneGameObjects)
```

Comment:
```suggestion
	for (GameObject* gameObject : sceneGameObjects)
```
I think that you are not using this index

Reply:
True, thank you!